go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/getsentry/sentry-go v0.9.0
 	github.com/giantswarm/apiextensions/v3 v3.16.1
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/exporterkit v0.2.0
+	github.com/giantswarm/exporterkit v0.2.1-0.20210204214054-ebd41525c595
```

Comment:
Will use released version when https://github.com/giantswarm/exporterkit/pull/31 is merged